packages/types/package.json

```diff
@@ -23,7 +23,7 @@
     "@types/pug": "2.0.6",
     "@types/serve-static": "1.15.2",
     "@types/terser-webpack-plugin": "4.2.1",
-    "@types/webpack": "4.41.33",
+    "@types/webpack": "^4.41.33",
```

Comment:
Maybe?

```suggestion
    "@types/webpack": "~4.41.33",
```

Reply:
Should be fine for now if you prefer that but could cause a similar issue in the future.

For reference, there seemed to be an issue with `@types/webpack-hot-middleware` before that broke types in a patch release (#18827) but I'm not aware of there being any issues with `@types/webpack`.

Reply:
@danielroe so would you prefer to use `~`? I'm a bit skeptical because then it's guaranteed to break once webpack types update the minor version. I'd rather trust and hope and they don't break types in minor revision.